syntax = "proto3";

message Address{
    string IP = 1;
    int32 port = 2;
}

service CommWithServer{
    rpc SendDetailsOfPR(Address) returns (SendDetailsOfPRResponse) {}
}

message SendDetailsOfPRResponse{
    string Status = 1;
}